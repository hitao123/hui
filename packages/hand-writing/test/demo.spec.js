describe('handwriting component', () => {
  it('should show handwriting', () => {

  });
});
