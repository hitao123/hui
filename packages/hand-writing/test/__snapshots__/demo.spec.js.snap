// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div>
  <div>
    <div class="h-hand-writing">
      <div class="h-hand-writing__container">
        <div class="h-hand-writing__main"><canvas id="canvas" class="h-hand-writing__canvas"></canvas></div>
      </div>
      <div class="h-hand-writing__rewrite"><button class="h-button h-button--default h-button--large">
          <!----> <span class="h-button__text">重写</span></button></div>
    </div>
  </div>
</div>
`;
